detox/android/detox/src/main/java/com/wix/detox/DetoxActionHandlers.kt

```diff
@@ -110,3 +110,61 @@ class QueryStatusActionHandler(
             })
         }
 }
+
+class RecordingStateActionHandler(
+        private val instrumentsManager: DetoxInstrumentsManager,
+        private val wsClient: WebSocketClient
+) : DetoxActionHandler {
+
+    override fun handle(params: String, messageId: Long) {
+        val recordingPath = JSONObject(params).optString("recordingPath", null)
+        if (recordingPath != null) {
+            instrumentsManager.startRecordingAtLocalPath(recordingPath)
+        } else {
+            instrumentsManager.stopRecording()
+        }
+
+        wsClient.sendAction("setRecordingStateDone", mapOf<String, Any>(), messageId)
```

Comment:
Can use `emptyMap` instead of `mapOf` here

Reply:
Agree